### ISSUE TITLE
KGAT-高阶传播

### ISSUE THREAD

**Issue by XYZOK000** [Userepic/KGAT-pytorch-master#4]
您好，我对高阶传播这部分代码有疑问，能不能麻烦您解释一下
        for idx, layer in enumerate(self.aggregator_layers):
            ego_embed = layer(ego_embed, self.A_in)
            norm_embed = F.normalize(ego_embed, p=2, dim=1)
            all_embed.append(norm_embed)
请问论文中的公式10就是ego_embed = layer(ego_embed, self.A_in)吗，那么论文中的邻居结点是如何体现高阶传播呢，怎么好像都是一阶节点在学习embedding呢。
期待您的回复。
